### ISSUE TITLE
ME3Explorer 5 Checklist

### ISSUE THREAD

**Issue by Mgamerz** [ME3Tweaks/LegendaryExplorer#194]
This is a list of things we need to do before releasing ME3Explorer 5.0.

## FEATURE FREEZE: OCTOBER 15
All new "features" or extensions to current features must be done on a new branch after October 15, as Beta will be merged into master for the release of ME3Explorer 5.0. After October 15 only bug fixes will allowed into Beta unless it is polishing off an existing feature.

### Things that need to be done prior to release
 -  [x] Memory leak checks + fixes @Mgamerz 
 -  [x] Improve new mesh renderer UI for camera positioning, draw distances @Mgamerz
 -  [x] Close #190 Sequence Editor handling of sequence imports
 -  [x] Close #188 Split ME3Explorer into UI + Library @Mgamerz
  - [x] Close #163 Handle package parsing exception better in AssetDB @Kinkojiro 
 -  [x] Close #167 Multithreaded package dumping hitting same file @Mgamerz 
  - [x] Close #189 AFC Compactor woes - @Mgamerz  **PENDING VERIFICATION**
  - [x] Close #193 Higher resolution main window assets - @ Audemus
  - [x] Close #203 Drop WPF user facing names
  - [x] Close #205 Update welcome pages for tools @Kinkojiro @SirCxyrtyx @Mgamerz 
  - [x] **ME3Explorer 4.1 -> 5.0 changelog** @Kinkojiro @SirCxyrtyx @Mgamerz 

Items not fixed for 5.0
 - Close #168 ME2 animations being borked up - Not sure this is fixable - may be umodel or udk. This won't be fixed in 5.0

## RELEASE CANDIDATES: NOVEMBER 1
Ideally release candidates should be shipping by November 1. NO NEW FEATURE POLISHING SHOULD BE DONE AFTER THIS DATE, ONLY CRITICAL BUGFIXES. The new build system has helped tremendously on getting new builds into users' hands so we already have a fairly stable codebase.
 - [x] Prepare release assets (screenshots)
 - [x] Prepare Nexus + ME3Tweaks pages
 - [ ] Example ME3ExplorerCore project + wiki page? Maybe something like, ME1 splash randomizer? (since Kismet stuff is not in the lib - would be easy and fast example, I think)

## TARGET SHIP DATE: NOVEMBER 6/7
I'm sure setting a hard deadline will work just fine in software development, nothing can go wrong.

**Comment by Mgamerz** [comment]
Question time: Should we axe ScriptDB? It's been listed as deprecated, I accidentally removed the icon (since I literally thought it was gone). It's an ancient tool that just lists where functions are stored, has a pretty ugly interface by our current standards, and is pretty much already superceded by Asset DB (though it doesn't do functions - it does classes). I'm in favor of axing it - I don't see it being useful, or even used, by anyone. We have only 2 other winforms apps that seem like they could be reworked (they have nearly no interface), but not for 5.0.
